Lib/gftools/util/google_fonts.py

```diff
@@ -459,3 +462,85 @@ def LicenseFromPath(path):
   """
   return _EntryForEndOfPath(path, _KNOWN_LICENSE_DIRS)
 
+
+def LoadLanguages(languages_dir):
+  languages = {}
+  for textproto_file in glob.iglob(os.path.join(languages_dir, '*.textproto')):
+    with open(textproto_file, 'r', encoding='utf-8') as f:
+      language = text_format.Parse(f.read(), fonts_pb2.LanguageProto())
+      languages[language.id] = language
+  return languages
+
+
+def LoadScripts(scripts_dir):
+  scripts = {}
+  for textproto_file in glob.iglob(os.path.join(scripts_dir, '*.textproto')):
+    with open(textproto_file, 'r', encoding='utf-8') as f:
+      script = text_format.Parse(f.read(), fonts_pb2.ScriptProto())
+      scripts[script.id] = script
+  return scripts
+
+
+def LoadRegions(regions_dir):
+  regions = {}
+  for textproto_file in glob.iglob(os.path.join(regions_dir, '*.textproto')):
+    with open(textproto_file, 'r', encoding='utf-8') as f:
+      region = text_format.Parse(f.read(), fonts_pb2.RegionProto())
+      regions[region.id] = region
+  return regions
+
+
+def SupportedLanguages(
+  ttFont,
+  languages=LoadLanguages(resource_filename("gftools", "lang/languages"))
```

Comment:
@m4rc1e are you planning to copy the lang metadata into gftools? I'm worried about the two views of lang metadata falling out of sync.

Reply:
For the time being yes.

We currently do this for the axis registry as well. In order to keep everything in sync, we have a [script](https://github.com/googlefonts/gftools/blob/main/update_reg.sh) which will download the relevant folder from google/fonts and add it here. We can do the same for the lang dir.

Ideally, we should have all of this stuff in a single repo that we can import into google/fonts, googlefonts/tools and fontbakery.